src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -233,7 +233,7 @@ export default class NavigationLibrary {
         let standardRouteWaypointModelList;
 
         // TODO: As amended, this may be an unsafe assumption. Needs to be reexaimed.
-        if (flightPhase === FLIGHT_PHASE.APRON) {
+        if (this.isAirborneFlightPhase(flightPhase)) {
```

Comment:
This makes me slightly nervous limiting it to conditions like these...

Also, don't you want the inverse condition here?